src/params/devp_leds_blink.c

```diff
@@ -0,0 +1,171 @@
+/**
+ * Platform LED blinking through DeviceParamaters.
+ *
+ * Copyright Thinnect Inc. 2021
+ * @license MIT
+ */
+#include "devp_leds.h"
+
+#include "devp.h"
+
+#include "platform.h"
+
+#include "cmsis_os2.h"
+
+#include <string.h>
+
+static uint8_t m_blink_leds;
+
+static uint32_t m_blink_on_ms = 0;
```

Comment:
Maybe init them as 1 to prevent accidental ultrafast blinking when timer is left default(0) for some reason?

Reply:
the timer callback checks that the argument is not 0 (which would be a fault anyway). Setting to 0 disables blinking.